smriprep/cli/run.py

```diff
@@ -324,6 +333,8 @@ def _warn_redirect(message, category, filename, lineno, file=None, line=None):
     # Precedence: --fs-license-file, $FS_LICENSE, default_license
     if opts.fs_license_file is not None:
         os.environ["FS_LICENSE"] = os.path.abspath(opts.fs_license_file)
+    elif os.path.exists(os.path.abspath("/opt/freesurfer/license.txt")):
+        os.environ["FS_LICENSE"] = os.path.abspath("/opt/freesurfer/license.txt")
```

Comment:
To keep the logic in the comment unchanged:

```suggestion
    default_license = os.path.join(os.getenv("FREESURFER_HOME", ""), "license.txt")
    if opts.fs_license_file is not None:
        os.environ["FS_LICENSE"] = os.path.abspath(opts.fs_license_file)
    elif "FS_LICENSE" not in os.environ and os.path.exists(default_license):
        os.environ["FS_LICENSE"] = default_license
```